images/hook/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/dumb-init /bin/sh
```

Comment:
we need dumb-init to handle signals properly:

https://github.com/Yelp/dumb-init#why-you-need-an-init-system